e2e/testsuite/testsuite.go

```diff
@@ -398,6 +398,7 @@ func (s *E2ETestSuite) AssertValidTxResponse(resp sdk.TxResponse) {
 	s.Require().NotEqual(int64(0), resp.GasWanted, respLogsMsg)
 	s.Require().NotEmpty(resp.Events, respLogsMsg)
 	s.Require().NotEmpty(resp.Data, respLogsMsg)
+	s.T().Logf("tx response: %s", resp)
```

Comment:
If we include this this will be quite verbose for all tests. I think the response logs being included should be enough. WDYT?

Reply:
yea...true. it was useful for checking the tx because checking that the tx is a valid response doesn't actually tell us if the tx succeeded or not when i was running the test locally, but we can verify the bank send transferred funds correctly as you indicated below.